src/main/java/Event.java

```diff
@@ -0,0 +1,21 @@
+public class Event extends Task{
+    private String from;
+    private String to;
+
+    public Event(String input) {
+        //"project meeting /from Mon 2pm /to 4pm"
+        super(input.split("/")[0].strip());
+        String[] tokens = input.split("/");
+        //"from Mon 2PM"
+        this.from = tokens[1].split("from")[1].strip();
+        this.to = tokens[2].split("to")[1].strip();
```

Comment:
Consider adding error handling here. What happens if an invalid event is created say `event something /from abc`, but is missing the `to`?